Tests/SonarQube.TeamBuild.PreProcessor.Tests/AnalysisConfigGeneratorTests.cs

```diff
@@ -23,64 +22,43 @@ public class AnalysisConfigGeneratorTests
         #region Tests
 
         [TestMethod]
-        public void AnalysisConfGen_CmdLinePropertiesOverrideFileSettings()
```

Comment:
This test is no longer required; the generator doesn't now aggregate the cmd line and file settings before writing them.